docusaurus/docs/Android/03-ui/06-guides/09-adding-custom-attachments-v2.mdx

```diff
@@ -0,0 +1,454 @@
+# Adding Custom Attachments V2
```

Comment:
Maybe it would be better to use the following format

- Adding Custom Attachments (`MessageInputView`)
- Adding Custom Attachments (`MessageComposerView`) 

Same goes for the menu options in docusaurus.

I know it's slightly verbose, but it's descriptive and it also automatically tells users which component this is aiming at, whereas V2 isn't particularly informative.